Source/Core/VideoBackends/Software/Clipper.cpp

```diff
@@ -331,7 +331,9 @@ void ProcessTriangle(OutputVertexData* v0, OutputVertexData* v1, OutputVertexDat
   }
 }
 
-static void CopyVertex(OutputVertexData* dst, const OutputVertexData* src, float dx, float dy,
+static const int LINE_PT_TEX_OFFSETS[8] = {0, 16, 8, 4, 2, 1, 1, 1};
```

Comment:
```suggestion
constexpr std::array<int, 8> LINE_PT_TEX_OFFSETS{0, 16, 8, 4, 2, 1, 1, 1};
```

Reply:
Note that this was copied from here:

https://github.com/dolphin-emu/dolphin/blob/0029ca84b07f44e3ce8c80a44fa0285a966b34d3/Source/Core/VideoCommon/GeometryShaderManager.cpp#L14

It also was originally an array of floats until 0ac710339100dc6dc2f1f4c094b372846decdce5; I don't think the concerns there apply to the software renderer.

Reply:
Sure, but this should still be migrated to `std::array`, since library implementations allow tuning/enabling bounds checks both in debug and release builds, allowing OOB accesses to be caught (and also forces decays to pointer to be explicit, but that doesn't apply here).